### PR TITLE
summarize-telemetry: skip bad log lines

### PR DESCRIPTION
**Pull request by dcposch**
@feross

### CODE REVIEW

File: bin/summarize-telemetry.js [hunk 1]
```diff
@@ -54,7 +54,7 @@ function die (err) {
   process.exit(1)
 }
 
-// Reads telemetry log files in parallel, then produces a summary array, one
+// oeads telemetry log files in parallel, then produces a summary array, one
```
Comment (feross) [inline bin/summarize-telemetry.js]: typo

File: bin/summarize-telemetry.js [hunk 2]
```diff
@@ -66,9 +66,20 @@ function loadTelemetrySummary (logFiles) {
...
         // Each log file contains one JSON record per line
+        console.log('Parsing ' + filename)
         try {
           var lines = json.trim().split('\n')
-          var records = lines.map(JSON.parse)
+          var records = lines
+            .map(function (line, i) {
+              try {
+                return JSON.parse(line)
+              } catch (e) {
```
Comment (feross) [inline bin/summarize-telemetry.js]: We usually use `err` for Error objects.

File: bin/summarize-telemetry.js (hunk 1 above)
Comment (dcposch) [inline bin/summarize-telemetry.js]: fixed

File: bin/summarize-telemetry.js (hunk 2 above)
Comment (dcposch) [inline bin/summarize-telemetry.js]: fixed

**Pull request merged**